app/src/main/java/org/oppia/app/splash/SplashFragment.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.app.splash
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.oppia.app.HomeActivity
+import org.oppia.app.R
+
+/** The SplashFragments navigates to Home page once the app is finished loading completely*/
+
+class SplashFragment : Fragment() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+  }
+
+  override fun onCreateView(
+    inflater: LayoutInflater, container: ViewGroup?,
+    savedInstanceState: Bundle?
+  ): View? {
+    val view = inflater.inflate(R.layout.splash_fragment, container, false)
+    routeToHomePage();
```

Comment:
Call this function onCreate() or remove onCreate(). Also, there's a semicolon in the function call :P

Reply:
resolved